### PR TITLE
Refactor export wp guide

### LINKED ISSUE #1998: Export an Existing WordPress Site Doc Update
Re: [Export an Existing WordPress Site](https://pantheon.io/docs/wordpress-export)
## Issue Description:

The following section no longer works. No more option to import a site from archive files. The new migration tool now uses Pantheon Migration plugin.

> Prepare Your Archive for Import
You now have three individual files that make one complete archive of your WordPress site. The import screen allows you to toggle between uploading your archive files or supplying a remote URL (e.g. Amazon S3, Dropbox, your existing server, etc.) to fetch the archives.

>You can upload each of your archive files separately, or package the archives within a single file. The max file upload import size is 100MB total. URL imports are limited to 500MB per input.

> Note: Modify Dropbox URLs to end in dl=1 instead of the default dl=0. This forces a download of your archive and avoids the Dropbox landing page.

## Suggested Resolution 

Remove this section.

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #1998 

## Effect
PR includes the following changes:
- Removes duplicate content from manual migration guide.
- Removes references to outdated import methods.
- Refactor the purpose of the guide, for those who are manually migrating and need to easily move files to their local computer for a manual migration.

## Remaining Work
- [x] Technical Review
- [x] Copy Review

### CODE REVIEW

File: source/_docs/wordpress-export.md [hunk 1]
```diff
@@ -5,15 +5,9 @@ tags: [export]
 categories: [wordpress]
 ---
 
-Follow the recommended process within [Migrate to Pantheon: WordPress](/docs/migrate-wordpress), which uses the Pantheon Migration plugin.
-
-The following scenarios are exceptions to the recommended process and require [manually migrating](https://pantheon.io/docs/migrate-manual/) the site:
```
Comment (alexfornuto) [inline source/_docs/wordpress-export.md]: This content lives on [here](https://pantheon.io/docs/migrate-manual/)

File: source/_docs/wordpress-export.md [hunk 2]
```diff
@@ -77,26 +72,18 @@ mysqldump -uUSERNAME -pPASSWORD DATABASENAME > $TARGET/db.sql
...
+## Files
 Export a tar.gz or .zip file of your files directory, which was intentionally omitted from the codebase import. These files are not tracked in Git; instead, they are stored in Valhalla, our network file system.
+
 ```php
 TARGET=~/Desktop
 SOURCE=~/Projects/mysite
 cd $SOURCE/wp-content/uploads
 tar -czf $TARGET/files.tar.gz .
 ```
-## Prepare Your Archive for Import
```
Comment (alexfornuto) [inline source/_docs/wordpress-export.md]: I tested and found this content to no longer apply.

File: source/_docs/wordpress-export.md (hunk 1 above)
Comment (rachelwhitton) [inline source/_docs/wordpress-export.md]: I definitely like avoiding dupe content, but in this case it helps people determine whether or not they're headed down the right track. Folks may land here and do unnecessary work, only to later learn they could have migrated with a few mouse clicks in the guided process.

Consolidating/simplifying our migration docs may be the ultimate solution, but that seems like work for another issue.

File: source/_docs/wordpress-export.md [hunk 3]
```diff
@@ -77,26 +72,18 @@ mysqldump -uUSERNAME -pPASSWORD DATABASENAME > $TARGET/db.sql
...
-
-You can upload each of your archive files separately, or package the archives within a single file. The max file upload import size is 100MB total. URL imports are limited to 500MB per input.
-
-
-<div class="alert alert-danger" role="alert">
-<h4 class="info">Warning</h4><p>Modify Dropbox URLs to end in <code>dl=1</code> instead of the default <code>dl=0</code>. This forces a download of your archive and avoids the Dropbox landing page.</p></div>
+## Next Steps
 
-Only provide one `.sql` file; if multiple are present the import will fail.
+Depending on your previous hosting platform, you can use tools like `rsync` or SFTP to copy these archive files locally. Follow either of the guides below to manually migrate your site.
```
Comment (rachelwhitton) [inline source/_docs/wordpress-export.md]: Can we revise this section to clearly point users towards successful import? Maybe something short, explaining that they'll need to finish the instructions on the dashboard by uploading and extracting the code & file archives via sftp, then importing the db in the dashboard tool. then direct to the manual doc for more context.

File: source/_docs/wordpress-export.md [hunk 4]
```diff
@@ -5,15 +5,9 @@ tags: [export]
...
 
-Follow the recommended process within [Migrate to Pantheon: WordPress](/docs/migrate-wordpress), which uses the Pantheon Migration plugin.
-
-The following scenarios are exceptions to the recommended process and require [manually migrating](https://pantheon.io/docs/migrate-manual/) the site:
-
-- You want to preserve the site's existing Git history
-- [WordPress Site Networks](/docs/wordpress-site-networks)
-- You can't install a plugin on your existing site (e.g. WordPress.com)
-- [Your site doesn't have a public URL (e.g. a local installation) and it's archive exceeds 500MB.](/docs/migrate-wordpress/#frequently-asked-questions)
+The process of [manually migrating](/docs/migrate-manual/) a WordPress site to Pantheon requires a copy of your site code, files, and database. Depending on the limitations of your previous host, you may find it helpful to create archives files to copy locally before importing to Pantheon. This page covers how to create those archives.
```
Comment (ari-gold) [inline source/_docs/wordpress-export.md]: Let's start by directing them to the main guided/automated path and then let them know if they still want to do manually to read on...

Comment (alexfornuto) [inline source/_docs/wordpress-export.md]: For sure. The idea is that this guide is for when you a) need to manually migrate, and b) need to get copies of your codebase db and files from another host.

File: source/_docs/wordpress-export.md (hunk 3 above)
Comment (alexfornuto) [inline source/_docs/wordpress-export.md]: @rachelwhitton that's covered by the manual migration guide. I think that this comment and the one on L10 can be solved by some better crosslinking, both on this guide and the manual migration one.

**Comment by rachelwhitton** [comment]
+1 

Aligned Drupal doc

File: source/_docs/migrate-troubleshooting.md [hunk 5]
```diff
@@ -142,7 +142,7 @@ If you'd like your existing Drupal site to get one-click updates from your [Cust
 
 ### What if I can't use drush on my existing Drupal site?
 
-As an alternative to `drush` you can use the [Backup and Migrate](/docs/drupal-export#create-archive-using-backup-and-migrate) module. Also see [manually create an archive](/docs/drupal-export#manually-create-archive).
+As an alternative to `drush` you can follow this doc to [manually migrate](/docs/drupal-export).your Drupal site.
```
Comment (rachelwhitton) [inline source/_docs/migrate-troubleshooting.md]: We shouldn't need to use our content types to reference other pages. 

Suggest: As an alternative to `drush` you can manually export and migrate. For details, see [Export an Existing Drupal Site](/docs/drupal-export).

**Pull request merged**